utils/ecc_test_vectors/generate_test_vectors.py

```diff
@@ -0,0 +1,421 @@
+import importlib
+import sys
+import argparse
+import json
+from fastecdsa.point import Point
+
+# Known endomorphisms of the curves
+endomorphism = {
+  "P256" : {
```

Comment:
Maybe we could remove those endomorphisms that are not being used

Reply:
This implies another commit in the elliptic curve library so we leave it as is for the moment